### ISSUE TITLE
SCM is broken in GOCART2G GEOSgcm

### ISSUE THREAD

**Issue by mathomp4** [GEOS-ESM/GEOSgcm_App#300]
I'm going to CC in a lot of people because I just try and keep the SCM running. I have no idea how to use it scientifically. 

cc: @narnold1 @wmputman @amdasilva @pcolarco @sdrabenh 

---

The SCM currently uses GOCART.data as AERO_PROVIDER. However, since https://github.com/GEOS-ESM/GEOSgcm_App/pull/264 removed the option for GOCART.data and did not introduce a GOCART2G.data, I no longer have any template on how one should set up for a data-driven GOCART run. When you run you get:
```
 ERROR: cannot find export AERO
pe=00000 FAIL at line=07054    MAPL_Generic.F90                         <unknown error>
pe=00000 FAIL at line=00638    MAPL_Generic.F90                         <status=-1>
pe=00000 FAIL at line=00556    MAPL_Generic.F90                         <status=-1>
PHYSICS::SetServices                          1494
```

As a blind attempt, I did try to just to set the `AERO_PROVIDER` to `GOCART2G` and edited `GEOS_ChemGridComp.rc` to look like the full model does by enabling GOCART, GOCART2G, ACHEM, HEMCO. Note that I'm not even sure one should *do* this for the SCM but... 🤷🏼 

Doing this, (and adding a symlink to `ExtData/chemistry`) at least one case of the SCM fails because of missing data. I tried the `arm_97jul` case and it fell apart in ExtData:
```
   EXTDATA: ---- 00080: pSO2_OCS
   EXTDATA: *******************************************************

pe=00000 FAIL at line=02143    ExtDataGridCompMod.F90                   <From ExtData/chemistry/MERRA2GMI/v0.0.0/L72/MERRA2_GMI.tavg24_3d_dac_Nv.x576_y361_t12.%y4.nc4 could not find time no extrapolation>
pe=00000 FAIL at line=01005    ExtDataGridCompMod.F90                   <status=1>
pe=00000 FAIL at line=01799    MAPL_Generic.F90                         <status=1>
pe=00000 FAIL at line=00767    MAPL_CapGridComp.F90                     <status=1>
```

The issue is, I think (I'll name drop @bena-nasa), that the MERRA2_GMI files are from 1999 on, but `arm_97jul` (as it says) is in 1997. So ExtData fails. It looks like many cases are not after 1999:
```
$ fd cap_restart -X cat | sort
19921101 000000
19921101 060000
19930715 000000
19930715 000000
19930715 000000
19930715 000000
19930715 000000
19930715 000000
19930715 000000
19930715 000000
19930715 000000
19930715 000000
19930715 000000
19930715 000000
19930715 000000
19930715 000000
19970618 000000
19970618 000000
19970618 233000
19970618 233000
19980501 000000
19980501 000000
19980507 210000
19980507 210000
```

At this point, a scientist on @GEOS-ESM/aerosol-team or someone needs to figure this out. Issues I see at least are:

1. Can you use GOCART2G with SCM or do you need to use data-driven?
   1. If full GOCART2G is possible, it looks like we might not have enough data to run at least some cases. 
   2. If data-driven GOCART2G, well, someone needs to provide an example of how to set one up since `gcm_setup` can no longer provide that. 
2. No matter what, someone then needs to make sure the model isn't outputting garbage.

Once all this is known, I can work on fixing up `scm_setup`, the input files, etc. to try and get it to work, but until then, the SCM is broken.

**Comment by pcolarco** [comment]
@mathomp4 I don't know what to tell you. Haven't run the SCM in 15+ years and don't know who is using it. I would defer to @amdasilva to prioritize this, but not clear I'm even capable of helping (let alone being able to do something on a useful time scale to you).

**Comment by sdrabenh** [comment]
@pcolarco the issue @mathomp4 mentioned is also true for any gcm run prior to 1999. We need `ExtData/chemistry/MERRA2GMI/v0.0.0/L72/MERRA2_GMI.tavg24_3d_dac_Nv.x576_y361_t12.*.nc4` files backfilled to 1979 in order to run the normal gcm. Do you have those files somewhere that Rob can copy?

**Comment by pcolarco** [comment]
OK, I misunderstood the issue. I should be able to produce those files. They derive from MERRA-2 GMI which goes back to 1980. I will "pad" to 1979 with the 1980 values and bring up to 1999. This will take some time as the files are large to produce.

**Comment by mathomp4** [comment]
@pcolarco Thanks. Hopefully that's all that's needed for it to run.

Well, maybe not run *scientifically* correct, but *doesn't crash* correct which is what I care about. 😄

**Comment by pcolarco** [comment]
I have pushed the MERRA2_GMI oxidant fields to $SHARE/dasilva/fvInput/MERRA2GMI/L72. They now span 1979 - 2025. Notes:
1979 is just a copy of 1980 since that is the first year of the MERRA2GMI run.
2020 and later are just copies of 2019 since that is the last year of the MERRA2GMI run.

I will ask Arlindo and Ravi to move around and copy onto higher vertical resolution files.

**Comment by mathomp4** [comment]
This is now "fixed" in #302 . Note that the SCM could be outputting garbage, but it runs.
